api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/BFFApiGatewayController.java

```diff
@@ -425,7 +425,7 @@ public Flux<VetDTO> getAllVets() {
         return vetsServiceClient.getVets();
     }
 
-    @IsUserSpecific(idToMatch = {"vetId"})
+    @SecuredEndpoint(allowedRoles = {Roles.ANONYMOUS})
```

Comment:
Hey Karina, I know that Dylan is the one who implemented the secure endpoints but just to double check, what is ANONYMOUS role?

Reply:
ANONYMOUS is anyone!